imaging/ml/ml_codelab/breast_density_auto_ml.ipynb

```diff
@@ -286,20 +283,19 @@
     "# Path to export DICOM data.\n",
     "dicom_store_url = os.path.join(HEALTHCARE_API_URL, 'projects', project_id, 'locations', location, 'datasets', dataset_id, 'dicomStores', dicom_store_id)\n",
     "path = dicom_store_url + \":export\"\n",
-    "\n",
     "# Headers (send request in JSON format).\n",
     "headers = {'Content-Type': 'application/json'}\n",
     "\n",
     "# Body (encoded in JSON format).\n",
-    "output_config = {'output_config': {'gcs_destination': {'uri_prefix': jpeg_folder, 'mime_type': 'image/jpeg; transfer-syntax=1.2.840.10008.1.2.4.50'}}}\n",
```

Comment:
Would be good to use 
gcloud beta healthcare dicom-stores export here if possible, but again, lets do this in a separate PR.

Reply:
https://github.com/GoogleCloudPlatform/healthcare/issues/433